db/compaction/compaction_job_test.cc

```diff
@@ -386,7 +386,7 @@ class CompactionJobTestBase : public testing::Test {
         kUnknownFileCreationTime,
         versions_->GetColumnFamilySet()->GetDefault()->NewEpochNumber(),
         kUnknownFileChecksum, kUnknownFileChecksumFuncName, kNullUniqueId64x2,
-        0, 0);
+        0, 0, true);
```

Comment:
Nit: would be nice to put the variable name as an inline `/* ... */` comment here